res/css/views/right_panel/_UserInfo.scss

```diff
@@ -76,33 +76,12 @@ limitations under the License.
         width: 160px;
     }
 
-    .mx_UserInfo_avatar {
-        margin: 24px 32px 0 32px;
-    }
-
-    .mx_UserInfo_avatar > div {
-        max-width: 30vh;
```

Comment:
this max-width was important too
and the specific max font-size too.

Before
![image](https://user-images.githubusercontent.com/2403652/120667275-3d1df500-c485-11eb-8ef8-16c04371a107.png)

After
![image](https://user-images.githubusercontent.com/2403652/120667298-427b3f80-c485-11eb-8a49-cc0ce453d0dc.png)